config/application.rb

```diff
@@ -10,8 +10,8 @@
 require "action_view/railtie"
 require "sprockets/railtie"
 require "active_storage/engine"
-# require "rails/test_unit/railtie"
 require_relative "../lib/gateways/email_gateway"
+require_relative "../lib/gateways/certificate_repository_s3"
```

Comment:
is this needed? Does the autoloader not load it?

Reply:
was in there for the email gateway already, so assumed needed